eiffel-syntax-and-usage/activity-linking.md

```diff
@@ -71,6 +71,23 @@ __Legal targets:__ [EiffelActivityTriggeredEvent][ActT],
 [EiffelTestCaseTriggeredEvent][TCT], [EiffelTestSuiteStartedEvent][TSS]  
 __Multiple allowed:__ Yes  
 
+### ORIGINAL_TRIGGER
+
+This link type is used for sequences of (usually) failing activities where the first activity declares the "root" trigger and the subsequent activities are retries of the first activity.
+
+*The image below shows how a source code change triggers a build activity. That activity fails and is retried, and the second activity (which might be manually triggered by a user) has an ORIGINAL_TRIGGER link to the first activity.*
+
+![An example of using ORIGINAL_TRIGGER](./original-trigger.png "An example of using ORIGINAL_TRIGGER")
+
+An ORIGINAL_TRIGGER link could be combined with a CAUSE link. For example, an activity that retries a failing build could conceivably have a CAUSE link to the ActF event that announces the failure, but an ORIGINAL_TRIGGER link would further increase the fidelity of the model by describing not just that the second activity is run after the first one but that the first activity is the original trigger of the activity sequence.
```

Comment:
Had to read this paragraph more than once to understand it. Could probably be easier with a picture but if not I have made an attempt to reformulate it (but do feel free to discard it)

```suggestion
To enhance clarity, an event could combine the ORIGINAL_TRIGGER with a CAUSE link. For example: An activity that retries failing builds, based on events, might use a CAUSE link to the ActF that that announces the failure. To increase the fidelity of the model the event could also use an ORIGINAL_TRIGGER, to the ActT, to signal which event was the original trigger of the activity sequence.
```

Reply:
Took you suggestion more or less as-in, just removing a stray word and some commas.